boards/posix/native_posix/doc/index.rst

```diff
@@ -270,6 +270,11 @@ The following peripherals are currently provided with this board:
   An optional UART driver can be compiled with native_posix.
   For more information refer to the section `UART`_.
 
+**Serial ports**
```

Comment:
I think we need to be a bit clearer about the fact that now we have 2 drivers, and the difference between both (one allows to connect to a pseudo-tty or stdin/out, the other to real attached devices).

Maybe a 
** UART/Serial ** 
heading followed by a very short description of the commonality difference
followed by the
*** ptty driver ***
*** tty driver ***
sections?

Reply:
Please look at the changed section. For the level three headers (ptty driver, tty driver) I used only two asterisks  on each side, as three asterisks would not render correctly. 

I also changed the headers of UART and Serial ports section. 

Does this look ok now?